packages/storage/test/testshared.ts

```diff
@@ -104,8 +107,9 @@ export function assertThrows(f: () => void, code: Code): FirebaseStorageError {
     }
   }).to.throw();
   expect(captured).to.be.an.instanceof(FirebaseStorageError);
+  // @ts-ignore Compiler thinks catch block is unreachable.
```

Comment:
I think it's because TS doesn't know that the callback is invoked immediately.

Hopefully they'll fix it at some point.

https://github.com/Microsoft/TypeScript/issues/11498

Reply:
Can you update the comment with the actual reason?

Reply:
Done.